src/screens/Menu/Menu.js

```diff
@@ -196,7 +205,12 @@ class Menu extends React.Component<Props, State> {
         hidden: !__DEV__,
       },
     ];
-  }
+
+    if (smartWalletFeatureEnabled) {
```

Comment:
Do we want to check Smart Wallet feature flag or Referrals?
```suggestion
    if (smartWalletFeatureEnabled) {
```